nautobot/core/settings.py

```diff
@@ -489,3 +489,23 @@
         "USE_REDIS_CACHE": "default",
     },
 }
+
+#
+# Celery (used for background processing)
+#
+
+# Instruct celery to report the started status of a job, instead of just `pending`, `finished`, or `failed`
+CELERY_TASK_TRACK_STARTED = True
+
+# Global task time limit
+CELERY_TASK_TIME_LIMIT = 30 * 60
```

Comment:
Should this (or any other Celery settings) be 1) exposed in `nautobot_config.py` and overridable via environment variables, 2) be documented in `optional-settings.md`?

Reply:
The time limit certainly should, and _maybe_ the broker and result URLs, but I think we need to play with those first.

Reply:
I'd like to add that since we're making this one user-configurable, let's please also add the environment variable check here, so that people don't have to re-generate their configuration or manually update it just to take advantage. (And also possibly cast it to an `int` if needed?)

```suggestion
CELERY_TASK_TIME_LIMIT = os.getenv("NAUTOBOT_CELERY_TASK_TIME_LIMIT", 30 * 60)
```